lib/elements/element.py

```diff
@@ -150,16 +150,26 @@ def set_param(self, name, value):
     @property
     @cache
     def style(self):
-        declarations = tinycss2.parse_declaration_list(self.node.get("style", ""))
-        style = {declaration.lower_name: declaration.value[0].serialize() for declaration in declarations}
+        return self.parse_style()
 
+    def parse_style(self, node=None):
```

Comment:
I think we want to add the `@cache` decorator so we're not parsing styles a bunch of times.